tests/layer_tests/tensorflow_tests/test_tf_UnaryOps.py

```diff
@@ -135,6 +136,7 @@ def create_net_with_unary_op(self, shape, ir_version, op_type, use_new_frontend)
 
     test_data_precommit = [dict(shape=[4, 6, 8, 10, 12])]
 
+    @pytest.mark.skipif(sys.version_info > (3, 10), reason="tensorflow_addons package is not available for Python 3.11 and higher")
```

Comment:
@rkazants FYI. Do you want me to create an issue or maybe do you have an idea for a workaround? The test is failing with ModuleNotFoundError. `tensorflow_addons` as of now doesn't support Python 3.11: https://pypi.org/project/tensorflow-addons/.

Reply:
Yes, please keep in touch to discuss this WA. I would like to have all activations functions tests get away.

Reply:
I suggest to replace `skip` with `xfail` so that you will be able to track when `tensorflow_addons` starts working

Reply:
xfail is rough solution. Let us handle it more accurately. Namely, move `mish` case to the separate case and import `tensorflow_addons` with handle `ImportError`

Reply:
Done, please take a look now.